src/overlays/actors/ovl_En_Ssh/z_en_ssh.c

```diff
@@ -37,139 +37,900 @@ const ActorInit En_Ssh_InitVars = {
+            EnSsh_SetColliderScale(this, 0.5f, 2.0f);
+        } else {
+            EnSsh_SetColliderScale(this, 0.75f, 2.0f);
+        }
+    }
+}
+
+void EnSsh_Start(EnSsh* this, GlobalContext* globalCtx) {
+    if (!EnSsh_IsCloseToGround(this)) {
+        EnSsh_SetupAction(this, EnSsh_Wait);
+        EnSsh_Wait(this, globalCtx);
+    } else {
+        EnSsh_SetLandAnimation(this);
+        this->stateFlags |= SSH_STATE_GROUND_START;
+        EnSsh_SetupAction(this, EnSsh_Land);
+        EnSsh_Land(this, globalCtx);
+    }
+}
+
+void EnSsh_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnSsh* this = THIS;
+
+    EnSsh_UpdateColliderScale(this);
+
+    if (EnSsh_CollisionCheck(this, globalCtx)) {
+        return;
+    }
+
+    if (this->stunTimer != 0) {
+        EnSsh_Damaged(this);
+    } else {
+        SkelAnime_Update(&this->skelAnime);
+        Actor_ApplyMovement(&this->actor);
+        Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 0.0f, 0.0f, 0.0f, 4);
+        this->actionFunc(this, globalCtx);
+    }
+
+    EnSsh_UpdateYaw(this, globalCtx);
+
+    if (DECR(this->blinkTimer) == 0) {
+        this->blinkTimer = Rand_S16Offset(60, 60);
+    }
+
+    this->blinkState = this->blinkTimer;
+    if (this->blinkState >= 3) {
+        this->blinkState = 0;
+    }
+
+    EnSsh_SetColliders(this, globalCtx);
+    Actor_SetHeight(&this->actor, 0.0f);
+}
+
+s32 EnSsh_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnSsh* this = THIS;
+
+    switch (limbIndex) {
+        case 1:
+            if ((this->spinTimer != 0) && (this->swayTimer == 0)) {
+                if (this->spinTimer >= 2) {
+                    EnSsh_AddBlureVertex(this);
+                } else {
+                    EnSsh_AddBlureSpace(this);
+                }
+            }
+            break;
+
+        case 4:
+            if (this->stateFlags & SSH_STATE_FATHER) {
+                *dList = object_ssh_DL_005850;
+            }
+            break;
+
+        case 5:
+            if (this->stateFlags & SSH_STATE_FATHER) {
+                *dList = object_ssh_DL_005210;
+            }
+            break;
+
+        case 8:
+            if (this->stateFlags & SSH_STATE_FATHER) {
+                *dList = object_ssh_DL_005F78;
+            }
+            break;
+    }
+    return false;
+}
+
+void EnSsh_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnSsh* this = THIS;
+
+    if ((limbIndex == 5) && (this->stateFlags & SSH_STATE_FATHER)) {
+        OPEN_DISPS(globalCtx->state.gfxCtx);
+
+        gSPDisplayList(POLY_OPA_DISP++, object_ssh_DL_0000D8);
+
+        CLOSE_DISPS(globalCtx->state.gfxCtx);
+    }
+    Collider_UpdateSpheres(limbIndex, &this->collider2);
+}
+
+void EnSsh_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    static TexturePtr D_80976178[] = { &object_ssh_Tex_001970, &object_ssh_Tex_001DF0, &object_ssh_Tex_0021F0 };
```

Comment:
I think you don't need the `&` there
```suggestion
    static TexturePtr D_80976178[] = { object_ssh_Tex_001970, object_ssh_Tex_001DF0, object_ssh_Tex_0021F0 };
```